internal/distro/fedora/distro.go

```diff
@@ -730,26 +731,27 @@ func (t *imageType) Manifest(customizations *blueprint.Customizations,
 	}
 	bp.Customizations = customizations
 
-	manifest, err := t.initializeManifest(bp, options, repos, nil, containers, seed)
+	manifest, warnings, err := t.initializeManifest(bp, options, repos, nil, containers, seed)
 	if err != nil {
-		return distro.Manifest{}, err
+		return distro.Manifest{}, warnings, err
```

Comment:
Not related to your changed, but this got me thinking, why are we returning `distro.Manifest{}` instead of `nil` if there was an error... 🤔 (probably a question for @achilleas-k)

Reply:
In addition, IMO a `nil` should be returned here, instead of the `warnings` if there was an error. This applies to all distros.

Reply:
> Not related to your changed, but this got me thinking, why are we returning `distro.Manifest{}` instead of `nil` if there was an error... thinking (probably a question for @achilleas-k)

I think in this case they're probably equivalent?  `nil` vs alias to empty byte slice.
I blame @teg :smile:

Reply:
Well, not really but almost... `[]byte{} != nil`

Reply:
Do you want me to do anything about this or are we good?

Reply:
I think that this is something on "us"... so no change needed from your end 😉